types.go

```diff
@@ -119,7 +119,7 @@ type xmlInfo struct {
 	XmlSchemaCollection string
 }
 
-func readTypeInfo(r *tdsBuffer) (res typeInfo) {
+func readTypeInfo(sess *tdsSession, r *tdsBuffer) (res typeInfo) {
```

Comment:
I'm not sure about the code-style. I did not want to copy the whole readTypeInfo func for v7.1 compatibility as the version-info is needed in readVarLen